### DISCUSSION TITLE
add/remove no go zones and/or invisible walls

### DISCUSSION THREAD

**Discussion by szbbalint** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#512] [Q&A]
Hi, 
Is it possible to add or remove no go zones or invisible walls on the card like in the xiaomi app.

**Comment by PiotrMachowski** [comment]
It's not possible at this moment

**Reply by szbbalint** [reply]
So that's why I can't find anything about it. And do you planned to add this (is it possible) to the integration?
Thanks for the fast reply :)

**Reply by PiotrMachowski** [reply]
Maybe someday ;) it should be possible to implement it for some vacuums
